docs/protos/kafmesh/discover/v1/discover_api.proto

```diff
@@ -0,0 +1,158 @@
+syntax = "proto3";
+
+package kafmesh.discover.v1;
+
+import "google/protobuf/duration.proto";
+
+option csharp_namespace = "Kafmesh.Discover.V1";
+option go_package = "discoverv1";
+
+service Discover {
+    rpc GetServiceInfo(DiscoverRequest) returns (DiscoverResponse);
+}
+
+message DiscoverRequest {
+}
+
+message DiscoverResponse {
+    string goStruct = 1;
+    string serviceName = 2;
+    string serviceDescription = 3;
+    Component component = 4;
+}
+
+message Component {
+    string name = 1;
+    string description = 2;
+    repeated Source sources = 3;
+    repeated Processor processors = 4;
+    repeated Sink sinks = 5;
+    repeated View views = 6;
+    repeated ViewSource viewSources = 7;
+    repeated ViewSink viewSinks = 8;
+    Persistence persistence = 9;
+}
+
+message Defaults {
+    int64 partition = 1;
+    int64 replication = 2;
+    string type = 3;
+    google.protobuf.Duration retention = 4;
+    google.protobuf.Duration segment = 5;
+}
+
+message DiscoverInfo {
+    string serviceName = 1;
+    string serviceDescription = 2;
+    Component component = 3;
+}
+
+message Input {
+    TopicDefinition topicDefinition = 1;
+}
+
+message Join {
+    TopicDefinition topicDefinition = 1;
+}
+
+message Lookup {
+    TopicDefinition topicDefinition = 1;
+}
+
+message MessageDefinitions {
```

Comment:
I don't think we will need these messages in discovery. Most of these settings are used for code gen.